pkg/kmm/kmm.go

```diff
@@ -125,6 +130,14 @@ func CleanUp(cfg Config, releaseLock bool, deleteAssets bool) (err error) {
 	return nil
 }
 
+func InstallNetwork(networkProvider string) (error) {
+	if np, err := network.CreateNetworkProvider(networkProvider); err != nil {
+		return err
```

Comment:
If there's an error here, does the network provider creation above need to be rolled back? (or is that just a local and transient thing?)